tutorials/jms-output-exactlyonce/README.md

```diff
@@ -0,0 +1,62 @@
+## JMS ActiveMQ Output Example
+
+
+This application shows the use of JMS output operator by implementing AbstractJMSSinglePortOutputOperator.
+The operator uses its default JMSTransactionalStore to ensure exactly once capability. AbstractJmsOutputOperator
+handles every window as a transaction. In every transaction the current windowId will be saved to a topic specific
+JMS meta-data queue to skip sending tuples of already committed windows.
+It is the responsibility of the user to create the meta-data queue in the JMS provider.
+
+The application generates a sequence of string numbers to be written to an ActiveMQ queue by a JMS output operator.
+The output operator is preceded by a intermediate pass-through operator which will intentionally fail after a
+specified amount of processed tuples, causing the operator to be redeployed and some tuples to be reprocessed.
+
+After that the ValidationApplication can be used to read and validate the queue for having no duplicates.
+It reads all messages from the queue, validates and writes the results to a HDFS validation file.
```

Comment:
Why is there a need to use HDFS here? This whole application/example uses mainly JMS and the validation file can very well be a local file system file. It can be HDFS too but why even mention HDFS here if it is completely unnecessary?